cve_bin_tool/checkers/bzip2.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for bzip2
+
+References:
+https://www.cvedetails.com/vulnerability-list/vendor_id-1198/product_id-2068/Bzip-Bzip2.html
+"""
+import re
+
+def guess_bzip2_version_from_content(lines):
+    """Guesses the bzip2 version from the file contents
+    """
+    new_guess = ""
+    patterns = [re.compile(r'([0-1]\.[0-1]\.[0-9]),\s+[0-9]{1,2}-[a-zA-Z]{3,4}-[0-9]{4}'),
+                re.compile(r'([0-1]\.[0-1]\.[0-9])'),
+                re.compile(r'([0-1]\.[0-9])'),
+                re.compile(r'([0-1]\.[0-1]\.[0-9])[a-z]'),
+                re.compile(r'([0-1]\.[0-9])\s+[A-Z]')
+                ]
```

Comment:
These patterns just make me nervous that we'd wind up with a lot of false positives.  I'd like to see this tested against some real world stuff that has bzip compiled in and also other libraries, but I can't think of any appropriate real-world test off the top of my head.  I think maybe some of the linux firmware stuff uses bzip compiled in, maybe?

Reply:
That would be great. 
bzip2 has been changing their version pattern frequently. https://github.com/enthought/bzip2-1.0.6. Also cve details website has bzip2 with differnt version patterns. So I had no choice but to include all version patterns.